lib/utils.js

```diff
@@ -166,6 +166,16 @@ async function setRealDeviceSecurity (keychainPath, keychainPassword) {
  * @property {string} platformVersion - The platform version of OS.
  * @property {string} platformName - The platform name of iOS, tvOS
 */
+
+/**
+ * Arguments for setting xctestrun file
+ * @typedef {Object} XctestrunFileArgs
+ * @property {DeviceInfo} deviceInfo - Information of the device under test
+ * @property {string} sdkVersion - The Xcode SDK version of OS.
+ * @property {string} bootstrapPath - The folder path containing xctestrun file.
+ * @property {number|string} wdaRemotePort - The remote port WDA is listening on.
+ * @property {string|undefined} wdaBindingIP - The IP address to bind to. If not given, it binds to all interfaces.
```

Comment:
since this argument is optional the declaration should look like `@property {string} [wdaBindingIP]`